src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -696,14 +696,32 @@ subroutine SoilStateInitTimeConst(bounds, soilstate_inst, nlfilename)
+    ! Initialize threshold soil moisture, and mass fraction of clay as
+    ! scaling coefficient of dust emission flux (kg/m2/s) in DUSTMod.F90
+    ! dmleung modified 5 Jul 2024, reducing sensitivity of dust emission
+    ! flux to clay fraction.
+    ! Also, for threshold soil moisture, dmleung followed Zender (2003)
+    ! DEAD scheme to again avoid dust flux being too sensitive to the choice
+    ! of clay dataset. This is different from what Leung et al. (2023)
+    ! intended to do.
+    ! Both decisions are based on tuning preference and could subject to
+    ! future changes. dmleung 5 Jul 2024
     ! --------------------------------------------------------------------
 
     do c = begc,endc
        g = col%gridcell(c)
 
-       soilstate_inst%gwc_thr_col(c) = 0.17_r8 + 0.14_r8 * clay3d(g,1) * 0.01_r8
+
+       ! dmleung++ added 24 May 2024: The below calculates the threshold gravimetric water content for the dust emission calculation in DUSTMod.F90. The equation comes from Eq. 14 of Fecan et al. (1999; https://doi.org/10.1007/s00585-999-0149-7).
+       !gwc_thr_col = 0.17*clay3d + 0.0014*(clay3d**2), and we only concern the topmost soil layer.
+       ! Charlie Zender later on added a tuning factor (a) such that the equation becomes gwc_thr_col = a*[0.17*clay3d + 0.0014*(clay3d**2)]. (Zender et al., 2003a; https://doi.org/10.1029/2002JD002775)
+       ! There are different options: E.g., a = 1 gives the first line below. E.g., Kok et al. (2014a, b) chose to use a = 1. A second choice is proposed by Charlie Zender (2003a):  a = 1/clay3d, which gives the second line below.
+       soilstate_inst%gwc_thr_col(c) = 0.17_r8 + 0.14_r8 * clay3d(g,1) * 0.01_r8 ! This was the original equation with a = 1 / (%clay) being the tuning factor for soil moisture effect in Zender's dust emission scheme. Danny M. Leung decided (Dec, 2023) that the Leung et al. (2023) dust emission scheme in the CESM will use Zender's tuning of a = 1 / (%clay), which overall encourages more dust emissions from seriamid and more marginal dust sources. Another advantage of using this tuning factor instead of a = 1 is that the dust emission threshold is linearly dependent on the clay fraction instead of parabolically dependent on clay fraction as in the above line. This means that dust emission becomes a little less sensitive to clay content (soil texture). 0.17 and 0.14 are fitting coefficients in Fecan et al. (1999), and 0.01 is used to convert surface clay fraction from percentage to fraction. dmleung added this detailed comment on 19 Feb 2024.
+
+       ! dust emission scaling factor dependent on clay fraction
        soilstate_inst%mss_frc_cly_vld_col(c) = min(clay3d(g,1) * 0.01_r8, 0.20_r8)
+       soilstate_inst%mss_frc_cly_vld_col(c) = 0.1_r8 + soilstate_inst%mss_frc_cly_vld_col(c) * 0.1_r8 / 0.20_r8   ! dmleung added this line to reduce the sensitivity of dust emission flux to clay fraction in DUSTMod. 5 Jul 2024
```

Comment:
@dmleung this new line means the returned value for when clay3d is zero -- will now be 0.1 rather than 0.2 as it was before. Is this intended here? Just wanted to make sure you were aware of that.

Reply:
I don't think so, I think with only line 722, when clay3d is zero, mss_frc_cly_vld_col will be zero. line 722 is used such that when clay3d > 0.2, mss_frc_cly_vld_col is capped at 0.2. ... I think my understanding is right? 

And then line 723 is here to limit mss_frc_cly_vld_col to vary between 0.1 and 0.2 instead of 0 to 0.2...

Our meeting just now actually reminded me one important thing: This factor will be used by Zender's scheme as well, so I probably should not do the change here, and instead do the change in DUSTMod inside the Leung scheme.

Reply:
The thing that caught it for me is a unit-test. But, you are right looking at the algebra the first line should be 0.2 and then the second line becomes

``` fortran
0.1_r8 + 0.2 * 0.1_r8 / 0.20_r8
```
which algebraically should be 

0.1 + 0.1 = 0.2

So I'll look into why the unit test isn't doing the right thing. I'll get back to you...

Reply:
Okay good. That could look a little complicated. I think it is best to write 
`0.1_r8 + mss_frc_cly_vld_col / 2.0_r8`
instead, if it's clearer. So, feel free to change it as you like...

Reply:
I did a little more work with the unit tester to clarify the behavior. I think I'll leave the form as it is. Basically for clay of 0 it's 0.1, at 10% 0.15, and then 0.2 once you get to 20% or higher. So it modifies the value for clay of 20% or less giving a minimum of 0.1.